app/assets/stylesheets/chats.scss

```diff
@@ -0,0 +1,29 @@
+.chat-table {
+  width: 350px;
+  margin: 0 auto;
+}
+
+.chat-table th, .chat-table td {
+    border-bottom: 1px solid #ddd;
+}
+
+.chat-receiver {
+  cursor: pointer;
+}
+
+.chat-topic {
+  margin-left: 228px;
+}
+
+.chat-table tr:hover {
+  background-color: #FFFFE0;
```

Comment:
See previous comment and try to apply it in the new stylesheets where colors appear.